testsuite/FixedLocalTuple/TestFixedLocalTupleList.py

```diff
@@ -1,6 +1,8 @@
-#  Copyright (C) 2017
+#!/usr/bin/env python2
+#
+#  Copyright (C) 2013-2017(H)
```

Comment:
2013 - why?

Reply:
when the paper of E++  appeared

Reply:
If a file was newly added, it should be the year of creation.